damage.py

```diff
@@ -21,6 +21,27 @@ def pre_mitigation_damage(
     return (base_damage + bonus_damage) * damage_modifier_percent_mult_factor + damage_modifier_flat
 
 
+def pre_mitigation_spell_damage(
+    base_spell_damage: float,
+    ratio: float,
+    base_damage: float,
+    bonus_damage: float,
```

Comment:
base_damage and bonus_damage are very confusing names to refer to base_AD, bonus_AD, base_AP, bonus_AP. There is no good name for this but even base_offense_stat, bonus_offense_stat would be easier to understand (atleast for me)